inc/classes/subscriber/preload/class-partial-preload-subscriber.php

```diff
@@ -140,8 +140,10 @@ public function preload_after_automatic_cache_purge( $deleted ) {
 						$file_path .= '/';
 					}
 				}
-
-				$this->urls[] = str_replace( $data['home_path'], $data['home_url'], $file_path );
+				$file_path = str_replace( $data['home_path'], '', $file_path );
```

Comment:
Good luck when `$data['home_path']` is equal to `/`, or when it is equal to `/foo` and `$file_path` is equal to `/foo/foobar/file.html` :s 
I think `preg_replace()` + `preg_quote()` will save us. Something like this (to test):
```php
$data['home_path'] = trailingslashit( wp_normalize_path( $data['home_path'] ) );
$file_path         = wp_normalize_path( $file_path );
$file_path         = preg_replace( '@^' . preg_quote( $data['home_path'], '@' ) . '@', '', $file_path );
$this->urls[]      = $data['home_url'] . $file_path;
```